src/TwigComponent/doc/index.rst

```diff
@@ -852,7 +859,7 @@ We already have a generic ``Alert`` component, so let's re-use it:
 .. code-block:: html+twig
 
     {# templates/components/Alert.html.twig #}
-    <div {{ attributes.defaults({ class: 'alert alert-'~ type }) }}">
+    <div {{ attributes.defaults({class: 'alert alert-'~ type}) }}">
```

Comment:
We have different conventions with the ~ , What do you think about not  adding space around it:

```suggestion
    <div {{ attributes.defaults({class: 'alert alert-'~type}) }}">
```

Reply:
Thanks!